tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -307,6 +314,12 @@ void DebuggerProbesInstrumentationRequester::AddMethodProbes(
             ProbesMetadataTracker::Instance()->CreateNewProbeIfNotExists(probeId);
         }
 
+        if (methodProbeDefinitions.empty())
+        {
+            Logger::Info("[AddMethodProbes] Early exiting, there are no new method probes to be added.");
```

Comment:
Should this be `DEBUG`?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.